.github/workflows/test-ci-cd.yml

```
@@ -0,0 +1,120 @@
+# GitHub Actions: 모든 서비스를 ECR로 Push + EC2에서 Pull & Run (with .env)
+name: CI/CD with GitHub Actions, Docker, and ECR
+
+on:
+  push:
+    branches: [ "test_all" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)